spring-cloud-kubernetes-client-config/pom.xml

```diff
@@ -80,6 +81,12 @@
 			<scope>test</scope>
 		</dependency>
 
+		<dependency>
```

Comment:
I would like to to be able to test k8s-client the same way we test fabric-8, which imho is very nice. As such, I need this dependency for `@Controller` and the like